@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -0,0 +1,989 @@
+                </div>
+                <div v-if="vmState.installMode === 'custom_config'" class="install-custom-config">
+                  <div>
+                    <UiTextarea v-model="vmState.cloudConfig" placeholder="Write configurations" accent="brand" href="''">
+                      {{ $t('new-vm.user-config') }}
+                    </UiTextarea>
+                    <span class="typo p3-regular-italic">
+                      Available template variables <br />
+                      - {name}: the VM's name. - It must not contain "_" <br />
+                      - {index}: the VM's index,<br />
+                      it will take 0 in case of single VM Tip: escape any variable with a preceding backslash (\)
+                    </span>
+                  </div>
+                  <div>
+                    <UiTextarea v-model="vmState.networkConfig" placeholder="Write configurations" accent="brand" href="''">
+                      {{ $t('new-vm.network-config') }}
+                    </UiTextarea>
+                    <span class="typo p3-regular-italic">
+                      Network configuration is only compatible with the NoCloud datasource. <br />
+
+                      See Network config documentation.
+                    </span>
+                  </div>
+                </div>
+                -->
+            </div>
+            <div v-else class="install-settings-container">
+              <div class="radio-container">
+                <UiRadioButton
+                  v-model="vmState.installMode"
+                  :disabled="!vmState.new_vm_template"
+                  accent="brand"
+                  value="ISO"
+                >
+                  {{ $t('new-vm.iso-dvd') }}
+                </UiRadioButton>
+                <UiRadioButton
+                  v-model="vmState.installMode"
+                  :disabled="!vmState.new_vm_template"
+                  accent="brand"
+                  value="PXE"
+                >
+                  {{ $t('new-vm.pxe') }}
+                </UiRadioButton>
+              </div>
+              <FormSelect v-if="vmState.installMode === 'ISO'" v-model="vmState.selectedVdi">
+                <template v-for="(VDIs, srName) in VDIsGroupedBySrName" :key="srName">
+                  <optgroup :label="srName">
+                    <option v-for="vdi in VDIs" :key="vdi.$ref" :value="vdi.$ref">
+                      {{ vdi.name_label }}
+                    </option>
+                  </optgroup>
+                </template>
+              </FormSelect>
+            </div>
+          </div>
+          <!-- SYSTEM SECTION -->
+          <UiTitle>{{ $t('new-vm.system') }}</UiTitle>
+          <!-- <UiToggle v-model="vmState.toggle">{{ $t('new-vm.multi-creation') }}</UiToggle> -->
+          <div class="system-container">
+            <div class="col-left">
+              <UiInput v-model="vmState.vm_name" accent="brand" href="''">{{ $t('new-vm.vm-name') }}</UiInput>
```

Comment:
```suggestion
              <UiInput v-model="vmState.vm_name" accent="brand" :label="$t('new-vm.vm-name')" href="''" />
```
Add links to doc for each input if needed.